README.md

```diff
@@ -104,6 +104,13 @@ class ListViewController: ViewController<ListView> {
 }
 ```
 
+Note that the `dispatch` method of view controllers is a bit different then the one exposed by the Katana store: it accept a simple `Dispatchable` and does not return anything. This is done to avoid implementing logic inside the view controller. If your interaction handler needs to do more than one single thing, you should pack all that logic in a side effect and dispatch that.
```

Comment:
```suggestion
Note that the `dispatch` method of view controllers is a bit different than the one exposed by the Katana store: it accepts a simple `Dispatchable` and does not return anything. This is done to avoid implementing logic inside the view controller. 

If your interaction handler needs to do more than one single thing, you should pack all that logic in a side effect and dispatch that.
```